.github/workflows/backend-e2e-test.yml

```diff
@@ -20,6 +20,11 @@ jobs:
         with:
           go-version: '1.13'
 
+      - name: download file Dockerfile-apisix
+        working-directory: ./api/test/docker
+        run: |
+          curl -o Dockerfile-apisix https://raw.githubusercontent.com/apache/apisix-docker/master/all-in-one/apisix/Dockerfile
```

Comment:
Not sure if it's ok to use the latest APISIX's Dockerfile 😳

Reply:
Thks, I may try the @imjoey method, so that I don't need to modify the yaml file, I just need to modify the docker compose file.
After modification, we only need to maintain the docker file of apisix-docker. At present, apisix Dockerfile is modified, and apiSix-dashboard Dockerfile-apisix also needs to be modified repeatedly.